class-sc-wp-updates-notifier.php

```diff
@@ -101,19 +100,23 @@ private function settings_up_to_date() {
 			if ( self::OPT_VERSION !== $current_ver ) { // is the version the same as this plugin?
 				$options  = (array) get_option( self::OPT_FIELD ); // get current settings from DB
 				$defaults = array( // Here are our default values for this plugin
-					'frequency'        => 'hourly',
-					'notify_to'        => get_option( 'admin_email' ),
-					'notify_from'      => get_option( 'admin_email' ),
-					'notify_plugins'   => 1,
-					'notify_themes'    => 1,
-					'notify_automatic' => 1,
-					'hide_updates'     => 1,
-					'notified'         => array(
+					'frequency'              => 'hourly',
+					'email_notifications'    => 0,
```

Comment:
I dont know if i should default to email_notifications to be on when the plugin is loaded for the first time or when someone upgrades to this version. My worry is that if someone upgrades the plugin but doesnt look at the settings page it will default email updates to off. For the long term i think it should default to off in general, but maybe this once, default it to on? IDK. Thoughts?

Reply:
I'd agree with this assessment.

Reply:
You could check what the previous version is and do different handling based on whether the person is upgrading or just installing.  It would seem best to preserve the previous settings for users who are updating.  For new (Alley) users, it would be better to have the person verify their email, etc, before email notifications are turned on... Since we want to receive the notifications and not the site owner.

Reply:
added code to deal with this